isis/CMakeLists.txt

```diff
@@ -68,7 +73,7 @@ option(isisTestData    "Directory containing IsisTestData"              OFF )
 option(testOutputDir   "Directory to store app test output folders"     OFF )
 option(buildCore       "Build the core ISIS modules"                    ON  )
 option(buildMissions   "Build the mission specific modules"             ON  )
-option(buildStaticCore "Build libisis3 static as well as dynamic"       OFF )
+option(buildStaticCore "Build libisis static as well as dynamic"       OFF )
```

Comment:
Nitpickiest thing ever, but could you put a space back in at the end of the string so that the OFFs and ONs line up again?